templates/csharp/custom-copilot-basic/.{{NewProjectTypeName}}/README.md.tpl

```diff
@@ -56,30 +56,29 @@ The app template is built using the Teams AI library, which provides the capabil
     ```
 {{/useAzureOpenAI}}
 1. In the debug dropdown menu, select Dev Tunnels > Create A Tunnel (set authentication type to Public) or select an existing public dev tunnel.
-2. Right-click the '{{NewProjectTypeName}}' project in Solution Explorer and select **Teams Toolkit > Select Microsoft 365 Account**
-3. Sign in to Teams Toolkit with a **Microsoft 365 work or school account**
+2. Right-click the '{{NewProjectTypeName}}' project in Solution Explorer and select **Microsoft 365 Agents Toolkit > Select Microsoft 365 Account**
+3. Sign in to Microsoft 365 Agents Toolkit with a **Microsoft 365 work or school account**
 4. Set `Startup Item` as `Microsoft Teams (browser)`.
 5. Press F5, or select Debug > Start Debugging menu in Visual Studio to start your app
 </br>![image](https://raw.githubusercontent.com/OfficeDev/TeamsFx/dev/docs/images/visualstudio/debug/debug-button.png)
 6. In the opened web browser, select Add button to install the app in Teams
 7. In the chat bar, type and send anything to your bot to trigger a response.
 
-> For local debugging using Teams Toolkit CLI, you need to do some extra steps described in [Set up your Teams Toolkit CLI for local debugging](https://aka.ms/teamsfx-cli-debugging).
+> For local debugging using Microsoft 365 Agents Toolkit CLI, you need to do some extra steps described in [Set up your Microsoft 365 Agents Toolkit CLI for local debugging](https://aka.ms/teamsfx-cli-debugging).
 
 ## Extend the AI Chat Bot template with more AI capabilities
 
 You can follow [Get started with Teams AI library](https://learn.microsoft.com/en-us/microsoftteams/platform/bots/how-to/teams%20conversational%20ai/how-conversation-ai-get-started) to extend the AI Chat Bot template with more AI capabilities.
```

Comment:
AI Chat Bot is not proper anymore. it's better to call it Agent.

...to extend the agent template with....

Reply:
I only comment this one file, but will require to update all other CEA/agents for Teams templates.